lib/alice/adapters.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Alice.Adapters do
+  @moduledoc false
+
+  def start_link(opts \\ [start_adapter: true]) do
+    if opts[:start_adapter], do: selected_adapter.start_link
+    {:ok, selected_adapter}
+  end
+
+  @doc """
+  Selects adapter, defaults to Slack
+  """
+  def selected_adapter do
+    Application.get_env(:alice, :adapter, Alice.Adapters.Slack)
```

Comment:
Elixir is awesome! Alice is definitely still development, but we've been using it in a few Slack instances for months now. The current release only supports Slack, but as you can see, we're doing a lot of work for the next 0.4 release (which might just become a 1.0 release).